app/models.py

```diff
@@ -2249,6 +2250,18 @@ def signed_agreements(self):
         return [a._asdict() for a in agreements]
 
 
+def check_for_uuid(data):
+    uuid_result = [False]
+    if data is None:
+        return uuid_result
+    if not data:
```

Comment:
not checks for anything that might resolve to false and None checks for NoneType